bootiso

```diff
@@ -29,6 +29,8 @@ set -E
 version="4.2.0"
 scriptName=$(basename "$0")
 bashVersion=$(echo "$BASH_VERSION" | cut -d. -f1)
+# Requres for any other distribution, like blackPanther OS
+DISTRO=$(lsb_release -si | awk '{print $1}'| tr [:upper:] [:lower:])
```

Comment:
Is there a  guarantee ` lsb_release`  would be available in any Linux system? I would rather have a graceful handling for when this command is missing, otherwise the console would be polluted.

Reply:
Lots of applications and scripts depends on lsb_release command for compatibility issues. So that shouldn't be a problem.

Reply:
I bet you're right. However, since one day this tool might be used on MacOS, I'd rather you replace this shell command with
```
lsb_release -si 2> /dev/null | awk '{print $1}'| tr [:upper:] [:lower:]
```